python/mxnet/ndarray.py

```diff
@@ -560,24 +559,6 @@ def wait_to_read(self):
         check_call(_LIB.MXNDArrayWaitToRead(self.handle))
 
 
-    @property
-    def shape(self):
```

Comment:
That means for any new function added to NDArray, we need to either implement it for SparseNDArray or disable it in SparseNDArray. Otherwise it confuses user with an inherited method which doesn't work